extra_docs/sensitivities/mean_sensitivity.tex

```diff
@@ -0,0 +1,33 @@
+\documentclass[11pt, oneside]{article}   	% use "amsart" instead of "article" for AMSLaTeX format
+\usepackage{geometry}                		% See geometry.pdf to learn the layout options. There are lots.
+\geometry{letterpaper}                   		% ... or a4paper or a5paper or ... 
+%\geometry{landscape}                		% Activate for rotated page geometry
+%\usepackage[parfill]{parskip}    		% Activate to begin paragraphs with an empty line rather than an indent
+\usepackage{graphicx}				% Use pdf, png, jpg, or eps§ with pdflatex; use eps in DVI mode
+								% TeX will automatically convert eps --> pdf in pdflatex		
+\usepackage{amsmath, amsthm, amssymb}
+\usepackage{macros}
+
+\begin{document}
+
+\begin{definition}
+The sample mean of database $X$ of size $n$ is 
+$$\bar{X} = \frac{1}{n} \sum_{i=1}^n x_i $$
+\end{definition}
+
+\begin{theorem}
+Say database $X$ has size $n$ and is bounded above by $M$ and bounded below by $m$. Then $\bar{X}$ has sensitivity bounded above by 
+$$ \frac{M-m}{n}.$$
+\end{theorem}
+
+\begin{proof}
+Say $X$ and $X'$ are neighboring databases which differ at data-point $x_j$. Then
```

Comment:
The distance metric used in this proof differs from the distance metric in the covariance proof. For lack of good words to describe it, this proof uses "edit," the other proof uses "update." Edit, as in, modifying a single record, and update, as in adding or removing a record.

Reply:
Here's a potential proof. I can update the latex if it looks right.
![mean_n+1](https://user-images.githubusercontent.com/3630029/65367399-fdef3800-dbfe-11e9-9fd3-ad1bd5a3b7b5.png)

Reply:
That looks correct to me. I think that if we add this version, we should also include the version where |X'| = |X|-1. I believe the proof is essentially the same, you just end up with n-1 in your denominator rather than n+1.

Reply:
I made a branch `MS_IGH_mean_distance` with a revised derivation. The library itself is not changed.
https://github.com/privacytoolsproject/PSI-Library/blob/MS_IGH_mean_distance/extra_docs/sensitivities/mean_sensitivity.pdf

Reply:
Hmm so the covariance proof is actually using the "edit" definition presented in the original mean sensitivity proof, not the "insertion/deletion" one. This is the same thing that Christian and I were grappling with in the variance proof last week. 

Basically the way both the covariance and variance proofs work is you're comparing two databases, which I call X' and X'', which differ in a single point. Call X the part of the database that is everything other than the point that differs. The proof then uses f(X) as part of the proof. But the actual two things being compared in the end are f(X') and f(X''). I think I need to change the wording to make this more clear though, as both you and Christian had trouble following that argument (as did I when rereading my own words). 

Having the alternate definition of neighboring that you suggest is fine, though I think we should be consistent throughout the library.